### PR TITLE
tox: Test on current Python

### PR DESCRIPTION
**Pull request by cclauss**
tox envlist = ~py27,~ py37,py38,py39,py310,py311

### CODE REVIEW

**Comment by sigmavirus24** [comment]
Let's drop 2.7 as well. Flake8 hasn't supported it for quite a while now

File: .github/workflows/tox.yml [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
+    branches: [master]
 jobs:
   tox-jobs:
     strategy:
       fail-fast: false
       matrix:
-        job: [py37-flake83]  # ,pypy,pep8,py3pep8,setuppy,manifest]
-    runs-on: ubuntu-latest
+        os: [ubuntu-latest]  # [macos-latest, ubuntu-latest, windows-latest]
+        python-version: ['3.7', '3.9', '3.11']  # ,pypy3.9,pep8,py3pep8,setuppy,manifest]
```
Comment (sigmavirus24) [inline .github/workflows/tox.yml:13]: Did I miss 3.10?

Comment (cclauss) [reply .github/workflows/tox.yml:13]: 3.8 and 3.10… The idea was to run the odd versions and not the even versions to save resources but if you think we need to _cover the waterfront_, we can do that.

Comment (cclauss) [reply .github/workflows/tox.yml:13]: Also, do we want to test on pypy3, run the manifest tests, etc?

Should we add pre-commit, black, mypy?  All optional.

**Review by sigmavirus24 — APPROVED** [review]

**Pull request merged**